sdk/identity/azure-identity-extensions/src/main/java/com/azure/identity/extensions/implementation/template/AzureAuthenticationTemplate.java

```diff
@@ -28,6 +34,10 @@ public class AzureAuthenticationTemplate {
 
     private AccessTokenResolver accessTokenResolver;
 
+    private IdentityCache<String, AccessToken> accessTokenCache;
```

Comment:
Let's not change the code in this class. Our goal is to keep this class as abstract as possible.